mrp_production_project_estimated_cost/models/product.py

```diff
@@ -20,8 +20,8 @@
 import openerp.addons.decimal_precision as dp
 
 
-class ProductTemplate(models.Model):
-    _inherit = 'product.template'
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
 
     manual_standard_cost = fields.Float(
```

Comment:
No. No es duplicado
Ya lo he explicado varias veces.

Si dejas solo el de odoo y pones por ejemplo un cálculo a precio medio, el sistema actualiza el valor. 
Hay clientes que SI quieren ese cálculo de coste medio pero necesitan un campo que pongan ellos algo y no se toque. Es para esto. 
Por otro lado, las Of ficticias, cuando suben un nuevo valor a la ficha de producto lo suben ahí, para no alterar el precio medio y poder hacer comparativas con lo calculado por el sistema y lo que se calcula "a pelo". Esto lo están usando ya por lo que tendría que mantenerse el campo.

Meter una tabla con todos los campos de coste, podría ser pero ahora mismo no lo ha pedido nadie y estamos desbordados por lo que de momento no le daría más vueltas. Sin más.

Saludos!!